cpu/cc2538/include/cpu_conf.h

```diff
@@ -20,6 +20,9 @@
 #define CPU_CONF_H
 
 #include "cpu_conf_common.h"
+/* disable -Wpedantic for vendor header */
+#pragma GCC diagnostic push
```

Comment:
hm .. I remember that we wanted to stay clear of pragmas as they hurt portability. Isn't it possible to disable the `-Wpedantic` flag via the `Makefile`? The `Makefile` solution would be a little more cumbersome, but at least our source code wouldn't contain commands / hints to the compiler.

Reply:
Compiler flags on the command line will not work for disabling the flag for only one header, not the entire compilation unit

Reply:
@gebart true, I didn't think that through.